starboard/android/shared/application_android.cc

```diff
@@ -251,7 +258,10 @@ void ApplicationAndroid::ProcessAndroidCommand() {
   JniEnvExt* env = JniEnvExt::Get();
   AndroidCommand cmd;
   int err = read(android_command_readfd_, &cmd, sizeof(cmd));
-  SB_DCHECK(err >= 0) << "Command read failed. errno=" << errno;
+  if (err < 0) {
+    SB_DCHECK(err >= 0) << "Command read failed. errno=" << errno;
```

Comment:
Can we monitor if this error happens on prod with Joel's UMA thing?

Reply:
@joeltine

Reply:
Telemetry relies on Chrome //base and //components/metrics libraries. As this is Starboard, which can't have those dependencies, I think the answer is no.